### PR TITLE
Add content to About

### LINKED ISSUE #42: Add content to the about page
A nice description about what we our aim is and how we try to achieve it would be nice.

### PR DESCRIPTION
**Pull request by rabuono**
closes #42

Still need to add link to Support page

### CODE REVIEW

File: about/index.md [hunk 1]
```diff
@@ -1,3 +1,27 @@
...
+ ## Funding acknowledgement
+Please check our [Support page](/about/support) for information on funding and contributing projects.
+ 
+## How to cite the IDTk
+Infectious Diseases Toolkit. A deliverable from the EU-funded BY-COVID project (grant agreement number 101046203). URL: https://www.infectious-diseases-toolkit.org/
+
+The Infectious Diseases Toolkit makes all of their materials publicly available under [Open Source Initiative](https://opensource.org/licenses) licenses.
+The process documents and data are made available under a [CC-BY](https://creativecommons.org/licenses/by/4.0/) license.
+Software are made available under an [MIT license](https://opensource.org/licenses/mit-license.html).
+For full details on licensing, please visit our [License document](https://github.com/elixir-europe/rdmkit/blob/master/LICENSE).
```
Comment (rabuono) [inline about/index.md:27]: Depends on issue #86 

```suggestion
For full details on licensing, please visit our [License document](https://github.com/elixir-europe/infectious-diseases-toolkit/blob/master/LICENSE).
```

**Comment by rabuono** [comment]
Closes #86

File: about/index.md [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
+General guidelines for data handling, analysis and visualisations in the context of infectious diseases. Guidelines are divided in categories and represent the contribution of communities from each domain.
+Tools, standards and overall resources that are contextually presented, including considerations for their need and use. 
+A collection of showcases. These are examples that combine and apply multiple resources (e.g. standards, tools, workflows) in the response to one or more infectious diseases.
+National resources pages in which country specific experiences, examples and resources are shared.
+The IDTk strives to connect users with the appropriate resources. A collection of tools, resources and training is offered, in addition to signposting external resources.
+ 
+ ## Funding acknowledgement
+Please check our [Support page](/about/support) for information on funding and contributing projects.
+ 
+## How to cite the IDTk
```
Comment (bedroesb) [inline about/index.md:21]: I don't know if this subtitle fits the content

Comment (rabuono) [reply about/index.md:21]: I found it a bit excessive given that there is a full page for support. But I can see that a portion of users might expect funding info on an about page. That is why I thought linking towards that page was better than repeating the information there.

Comment (bedroesb) [reply about/index.md:21]: I am ok with the content of the subheading, but based on the subtitle I would expect an answer in how I should cite IDTk, but instead it talks about licensing and funding :)

Comment (rabuono) [reply about/index.md:21]: I misread the comment :facepalm:, sorry

The citing is so far just attached to the url and the project context in which IDTk is being built, as there is no publication or identifier.

We might add a '##License' subheading above the licensing info. Would that solve it?

**Review by hedi-ee — APPROVED** [review]

File: about/index.md (hunk 2 above)
Comment (rabuono) [reply about/index.md:21]: Adding citation box to separate content

**Pull request merged**